Tools/Scripts/update-webkit-libs-jhbuild

```diff
@@ -49,15 +49,15 @@ GetOptions(
     'debug' => \$debug
 );
 
-sub getJhbuildIncludedFilePaths
+sub addJhbuildIncludedFilePaths
 {
-    my $jhbuildFile = shift;
+    my ($jhbuildFile, $includes) = @_;
```

Comment:
Should '$includes' be an array (@includes) instead of an scalar? (I think as it's now '$includes' is reference to an array, so it may work as well).

Reply:
I've reworked the patch. This subroutine doesn't take `includes` as an argument anymore.